corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -511,9 +513,19 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                             popupText += "<b>" + headerSanitized + ":</b> " + valueSanitized + "<br>";
                         }
                     });
+                    const rowId = `row-${model.id}`;
                     L.marker(latLng)
                         .addTo(addressMap)
-                        .bindPopup(popupText);
+                        .bindPopup(popupText)
+                        .on('click', () => {
```

Comment:
Can this add/remove a class instead of manipulating styles directly? Then the actual styling could live in LESS (the color hex code in particular caught my eye).

Reply:
For sure

Reply:
Sure - my thoughts on the color are just that it should be something in the CommCare palette, and it should be visible.